lib/components/link.js

```diff
@@ -85,19 +85,29 @@ export const computed = {
     }
 };
 
-function computeTag(props) {
-    return props.to && !props.disabled ? "router-link" : "a";
+function computeTag(props, parent) {
+    return Boolean(parent.$router) && props.to && !props.disabled ? "router-link" : "a";
 }
 
 function computeHref({ disabled, href, to }, tag) {
+    // We've already checked the parent.$router in computeTag,
+    // so router-link means live router.
     // When deferring to Vue Router's router-link,
     // don't use the href attr at all.
-    if (tag === "router-link") return null;
+    // Must return undefined for router-link to populate href.
+    if (tag === "router-link") return void 0;
     if (disabled) return "#";
```

Comment:
Do we need to set this to `#` now when disabled?

Reply:
I personally don't see the reason for this either. I think that ideally the link value wouldn't change while disabled.